src/components/Home/Keshav/Week5/redux/store/store.ts

```diff
@@ -0,0 +1,6 @@
+import { createStore } from 'redux';
+import CounterReducer from '../reducers/CounterReducer';
+
+const store = createStore(CounterReducer);
+
```

Comment:
create store is deprecated now, try using configure store

Reply:
I have used 'redux' for my week-5 assignment. "configureStore()" function is provided by "redux toolkit".